src/api/index.js

```diff
@@ -17,3 +17,4 @@ export { default as subscriptionAdd } from './subscriptionAdd';
 export { default as subscriptionRemove } from './subscriptionRemove';
 export { default as sendMessage } from './sendMessage';
 export { default as uploadFile } from './uploadFile';
+export { default as getRawMessage } from './getRawMessage';
```

Comment:
`getSingleMessage` is current version. Makes more sense than my previous versions